extensions.json

```
@@ -244,8 +244,8 @@
     {
       "id": "christianvoigt.argdown-vscode",
       "repository": "https://github.com/christianvoigt/argdown",
-      "version": "1.5.3",
-      "checkout": "argdown-vscode@v1.5.3",
+      "version": "1.7.0",
+      "checkout": "@argdown/cli@v1.7.0",
```

Upgrade script guessed the wrong tag:

```suggestion
      "checkout": "argdown-vscode@1.7.0",
```